org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WChosenboxListEditor.java

```diff
@@ -254,16 +256,18 @@ private void init()
         {
             popupMenu = new WEditorPopupMenu(false, true, isShowPreference(), false, false, false, lookup);
     		addChangeLogMenu(popupMenu);
-
-    		Menuitem editor = new Menuitem();
-    		editor.setAttribute("EVENT", WEditorPopupMenu.ASSISTANT_EVENT);
-    		editor.setLabel(Msg.getMsg(Env.getCtx(), "Assistant"));
-    		if (ThemeManager.isUseFontIconForImage())
-    			editor.setIconSclass("z-icon-Wizard");
-    		else
-    			editor.setImage(ThemeManager.getThemeResource("images/Wizard16.png"));
-    		editor.addEventListener(Events.ON_CLICK, popupMenu);
-    		popupMenu.appendChild(editor);
+    		
+    		if (gridField.getDisplayType() == DisplayType.ChosenMultipleSelectionList) {
+        		Menuitem editor = new Menuitem();
+	        		editor.setAttribute("EVENT", WEditorPopupMenu.ASSISTANT_EVENT);
+        		editor.setLabel(Msg.getMsg(Env.getCtx(), "Assistant"));
+        		if (ThemeManager.isUseFontIconForImage())
+        			editor.setIconSclass("z-icon-Wizard");
+        		else
+        			editor.setImage(ThemeManager.getThemeResource("images/Wizard16.png"));
+        		editor.addEventListener(Events.ON_CLICK, popupMenu);
+        		popupMenu.appendChild(editor);    			
+    		}
```

Comment:
I don't understand, why do need to add this check in the Chosenbox List Editor ?

Reply:
Otherwise, the assistant is available for TableDir editors - but content is empty as the panel is working with ref lists only

Reply:
this doesn't make sense to me, how can a method in WChosenboxListEditor effect the behaviour of WTableDirEditor ?

Reply:
Arff, sorry, not TableDir, is MultipleSelectionTable (which I called _MultipleTableDir_)

Reply:
done

Reply:
hi @nmicoud , the description of this PR and the comments at the Jira ticket still say "TableDir", that MUST be changed too.

Reply:
oops, done